src/core/default-repo/index.js

```diff
@@ -1,7 +0,0 @@
-'use strict'
-
-const isNode = require('detect-node')
-
-module.exports = isNode
-  ? require('./node')
-  : require('./browser')
```

Comment:
super sweet that we can get webpack, browserify and others to pick up browser vs node files through package.json 👍